transforms/images/polus-ftl-label-plugin/Dockerfile

```diff
@@ -1,28 +1,42 @@
 # Compile ftl.pyx
 FROM python:3.9
 ARG EXEC_DIR="/opt/executables"
-RUN mkdir -p ${EXEC_DIR}
+RUN mkdir -p ${EXEC_DIR}/executables
 COPY src ${EXEC_DIR}/.
 WORKDIR ${EXEC_DIR}
 RUN pip3 install --upgrade cython==3.0a6 \
-    && pip3 install numpy==1.19.1 --no-cache-dir \
+    && pip3 install numpy==1.20.0 --no-cache-dir \
     && python3 setup.py build_ext --inplace
+RUN cp executables/* . && rm -rf executables
 
 # Build the plugin container
 FROM python:3.9-slim
 
-COPY VERSION /
-		
-ARG EXEC_DIR="/opt/executables"
+# Get essential packages
+RUN apt-get update && \
+    apt-get install -y build-essential curl && \
+    apt-get update
```

Comment:
I know this PR was submitted before we had this in place, but we should add in the environment variables `POLUS_LOG` and `POLUS_EXT`. We should also update the code to use them.

Reply:
I added the variables and am using them in code.